wwwroot/index.html

```diff
@@ -338,14 +338,21 @@ <h2 class="mb-5">Frequently Asked Questions</h2>
                             </div>
                         </div>
                     </div>
-                    <div class="col-lg-4">
+                    <div class="col-lg-5">
                         <div class="sticky-top pt-header">
-                            <div class="box" id="addtowasabi">
+                            <strong class="text-primary mb-2 d-block lh">We strongly recommend using the Ginger Wallet client for long-term reliability!</strong>
+                            <a href="#download" class="btn btn-primary mb-4">Download</a>
+                            <div class="box" id="addtobtcpay">
+                                <h4 class="mb-4">What is Ginger Coordinator address?</h4>
+                                <strong>https://api.gingerwallet.io/</strong>
+                            </div>
+
+                            <div class="box mt-4" id="addtowasabi">
                                 <h4 class="mb-4">How to add Ginger coordinator to Wasabi Wallet?</h4>
                                 <ol class="text-start">
-                                    <li>Make sure that you are using the latest Wasabi Wallet version.</li>
-                                    <li>Go to Settings -> Bitcoin tab -> Coordinator URI</li>
-                                    <li>Change https://api.wasabiwallet.io/ -> https://api.gingerwallet.io/</li>
+                                    <li>Make sure that you are using Wasabi Wallet 2.0.7.2 or earlier version.</li>
+                                    <li>Click to Search bar » Select Data folder » Find Config.json</li>
```

Comment:
Click to Search bar » Type "Config File" » Press ENTER

Reply:
Done.